podpac/core/coordinates/coordinates.py

```diff
@@ -866,7 +884,14 @@ def unique(self):
             New Coordinates object with unique, sorted coordinate values in each dimension.
         """
 
-        return Coordinates([c[np.unique(c.coordinates, return_index=True)[1]] for c in self.values()])
+        coords = list(self.values())

```

Comment:
What happened here? I'm curious what didn't work about the previous version.

Reply:
Ahh, I see, the indexing is removed. I think you need to put that back, and then you can revert this.